### PR TITLE
find token by address and chainId

### LINKED ISSUE #2241: Matching pool funds set as FTM but display as ETH
**Overview**
You can see in this video that I set the matching pool funds in FTM, but they're displaying as "ETH" in Round Settings.

**Steps To Reproduce**
1. Watch this video: https://www.loom.com/share/1f464e66816748e0855383804f4de866
2. Open the round: https://rmgitcoin.on.fleek.co/#/round/0xc9293eca90ddc802f12dfea2d412a959aa4264a4?debug=true
3. Go to round settings -> Funding settings
4. Observe that matching pool is denominated in ETH

### PR DESCRIPTION
**Pull request by gravityblast**
closes #2241

### CODE REVIEW

File: packages/round-manager/src/features/round/FundContract.tsx [hunk 1]
```diff
@@ -92,7 +92,9 @@ export default function FundContract(props: {
     props.round &&
     payoutTokens.filter(
       (t) =>
-        t.address.toLocaleLowerCase() == props.round?.token?.toLocaleLowerCase()
+        t.address.toLocaleLowerCase() ==
```
Comment (bard) [inline packages/round-manager/src/features/round/FundContract.tsx:95]: Not introduced by this PR but I think we should use `toLowerCase` here, locale isn't relevant for chain addresses.

File: packages/round-manager/src/features/round/FundContract.tsx [hunk 2]
```diff
@@ -92,7 +92,9 @@ export default function FundContract(props: {
...
+          props.round?.token?.toLocaleLowerCase() &&
+        t.chainId == props.round?.chainId
```
Comment (bard) [inline packages/round-manager/src/features/round/FundContract.tsx:97]: Please use strict equality `===`

**Review by bard — CHANGES_REQUESTED** [review]

File: packages/round-manager/src/features/round/FundContract.tsx (hunk 1 above)
Comment (gravityblast) [reply packages/round-manager/src/features/round/FundContract.tsx]: @bard yes it makes sense I updated it

**Comment by gravityblast** [comment]
@bard I pushed now

**Review by bard — DISMISSED** [review]

**Review by bard — APPROVED** [review]

**Review by bard — APPROVED** [review]

**Review by vacekj — APPROVED** [review]

**Pull request merged**